scenes/enumerated/problem_10384/problem_10384.py

```diff
@@ -0,0 +1,243 @@
+from manim import *
+from lib.aramanim import *
+from lib.qarakusiscene import TaskNumberBox
+from objects import DScissors, Checkmark
+from .text import *
```

Comment:
Can you avoid `import *`-s. In case of conflicts, something unexpected or unwanted can happen.